Streams/FileStreamWriter.h

```diff
@@ -17,9 +18,14 @@ class FileStreamWriter : public SeekableStreamWriter
 	void Seek(uint64_t offset) override;
 	void SeekRelative(int64_t offset) override;
 
+	inline std::string GetFilename() const {
```

Comment:
Maybe this should return a const reference. It's likely to be used in a read-only manner, so that would avoiding copying to a new `std::string` object, which this currently does.

Reply:
Agreed. I hadn't thought of this.